src/Raven.Server/RavenServer.cs

```diff
@@ -815,10 +815,24 @@ private void SendErrorIfPossible(TcpConnectionOptions tcp, Exception e)
             return stream;
         }
 
+
+
         private bool TryAuthorize(RavenConfiguration configuration, Stream stream, TcpConnectionHeaderMessage header, out string msg)
```

Comment:
I was trying to avoid sending two messages.
I'll see if we actually do anything with the response, i would still want to use this message and not have to use two messages maybe just adding a Boolean stating the type of the error